Update seventeentrack config (#22428)
home-assistant/home-assistant.github.io: source/_integrations/seventeentrack.markdown

  - Postal Service
ha_release: 0.83
ha_iot_class: Cloud Polling
ha_config_flow: true
ha_domain: seventeentrack
ha_platforms:
  - sensor
ha_integration_type: integration
---

> The `seventeentrack` sensor platform allows users to get package data tied to their [17track.net](https://api.17track.net/en) account. The platform creates both summary sensors, which show the number of packages in a current state (e.g., "In Transit"), as well as individual sensors for each package within the account.

Reviewer: Just noticed this one, as this is an integration now :)

```suggestion
The 17TRACK integration allows users to get package data tied to their [17track.net](https://api.17track.net/en) account. The platform creates both summary sensors, which show the number of packages in a current state (e.g., "In Transit"), as well as individual sensors for each package within the account.
```

## Getting Your API Token

> 1. Register for an account: https://user.17track.net/en/register

Reviewer: Please don't use bare URLs.

2. Enter the verification code sent to your email.

> 3. Select "Developer Account"

Reviewer: ```suggestion
3. Select "Developer Account".
```

4. Complete registration, phone number and skype are not required.

> 6. Open the settings page: https://api.17track.net/en/admin/settings

Reviewer: Please don't use bare URLs.

> 7. API Token will be displayed under the Security tab

Reviewer: ```suggestion
7. API Token will be displayed under the Security tab.
```

> {% include integrations/config_flow.md %}

Reviewer: Did you forget to update the frontmatter to indicate config flow support?
